docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
+        the passed time interval. This method can be chained.<br /><br />
+        
+        If warpBoolean is true, additional [page:.warp warping] (gradually changes of the time scales) 
+        will be applied.<br /><br />
+        
+        Note: Like with *fadeIn*/*fadeOut*, the fading starts/ends with a weight of 1.
+        
+    </div>
 
-    <h2>Methods</h2>
+    <h3>[method:AnimationAction crossFadeTo]( [page:AnimationAction fadeInAction], [page:Number durationInSeconds], [page:Boolean warpBoolean] )</h3>
+    <div>
+        Causes this action to [page:.fadeOut fade out], fading in another action simultaneously, within 
+        the passed time interval. This method can be chained.<br /><br />
+        If warpBoolean is true, additional [page:.warp warping] (gradually changes of the time scales) 
+        will be applied.<br /><br />
+        
+        Note: Like with *fadeIn*/*fadeOut*, the fading starts/ends with a weight of 1.
+    </div>    
+
+    <h3>[method:AnimationAction fadeIn]( [page:Number durationInSeconds] )</h3>
+    <div>
+        Increases the [page:.weight weight] of this action gradually from 0 to 1, within the passed time 
+        interval. This method can be chained.
+    </div>
 
+    <h3>[method:AnimationAction fadeOut]( [page:Number durationInSeconds] )</h3>
+    <div>
+        Decreases the [page:.weight weight] of this action gradually from 1 to 0, within the passed time 
+        interval. This method can be chained.
+    </div>
+    
+    <h3>[method:Number getEffectiveTimeScale]()</h3>
+    <div>
+        Returns the effective time scale (considering the current states of warping and 
+        [page:.paused paused]).
+    </div>
 
-    <h3>[method:AnimationAction play]()</h3>
+    <h3>[method:number getEffectiveWeight]()</h3>
+    <div>
+        Returns the effective weight (considering the current states of fading and 
+        [page:.enabled enabled]).
+    </div>
 
-    <h3>[method:AnimationAction stop]()</h3>
+    <h3>[method:AnimationClip getClip]()</h3>
+    <div>
+        Returns the clip which holds the animation data for this action.
+    </div>
 
-    <h3>[method:AnimationAction reset]()</h3>
+    <h3>[method:AnimationMixer getMixer]()</h3>
+    <div>
+        Returns the mixer which is responsible for playing this action.
+    </div>
+
+    <h3>[method:Object3D getRoot]()</h3>
+    <div>
+        Returns the root object on which this action is performed.
+    </div>    
+    
+    <h3>[method:AnimationAction halt]( [page:Number durationInSeconds] )</h3>
+    <div>
+        Decelerates this animation's speed to 0 by decreasing [page:.timeScale timeScale] gradually 
+        (starting from its current value), within the passed time interval. This method can be chained.
+    </div>    
 
     <h3>[method:Boolean isRunning]()</h3>
+    <div>
+        Returns true, if this action is not only activated in the mixer, but really playing (as opposed 
```

Comment:
>Returns true, if this action is not only activated in the mixer, but really playing (as opposed...

This might read better as

> Returns true if this action is  activated in the mixer and currently playing (as opposed 

Also unnecessary comma after returns true 😛